ci-operator/config/openshift-priv/installer/openshift-priv-installer-release-4.19.yaml

```diff
@@ -577,7 +577,7 @@ tests:
   steps:
     cluster_profile: gcp-qe
     env:
-      FEATURE_GATES: '["GCPClusterHostedDNS=true"]'
+      FEATURE_GATES: '["GCPClusterHostedDNSInstall=true"]'
```

Comment:
https://github.com/openshift/api/pull/2423 is only applied to 4.20, rather than 4.19, right ?

Reply:
Thanks for the catch. Yes, you're right. We should only update this for 4.20+

/lgtm cancel

Reply:
Thanks @jianlinliu. I am still stuck in 4.19! Updated. https://github.com/openshift/installer/pull/9854 have also merged.